README.md

```diff
@@ -182,6 +182,51 @@ related:
 ```
 
 
+## components.yaml
```

Comment:
You don't mention anywhere this is for organizing code samples, might not be obvious for everybody.